lib/iex/test/iex/helpers_test.exs

```diff
@@ -656,6 +700,15 @@ defmodule IEx.HelpersTest do
 
       assert capture_io(fn -> b(Exception.message() / 1) end) ==
                "@callback message(t()) :: String.t()\n\n"
+
+      assert capture_io(fn -> b(:gen_server.handle_cast() / 2) end) == """
```

Comment:
The format imposed by the formatter here is a bit weird, but I think nothing can be done here.